packages/playground/remote/src/lib/config.ts

```diff
@@ -1,3 +1,3 @@
 export const wordPressSiteUrl = new URL('/', (import.meta || {}).url).origin;
 // Hardcoded in wp.js:
-export const DOCROOT = '/wordpress';
+export const DOCROOT = '/wordpress/build';
```

Comment:
This was changed so it works for wp-develop. 

Ideally the docroot is changed dynamically depending on the WP version, or, alternatively, the setup is changed in a way that the docroot is always the same.

Reply:
[`esm-prefix.js`](https://github.com/WordPress/wordpress-playground/blob/trunk/packages/playground/compile-wordpress/build-assets/esm-prefix.js) exists specifically for this use-case. It could be expanded with something like:

```ts
export const docRoot = WP_DOCROOT;
```

And then the Dockerfile would provide the string value around here:

https://github.com/WordPress/wordpress-playground/blob/da38192af57a95699d8731c855b82ac0222df61b/packages/playground/compile-wordpress/Dockerfile#L192-L197